sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileStore.java

```diff
@@ -70,46 +86,54 @@ public boolean isReadOnly() {
     }
 
     /**
+     * Containers do not limit the amount of data stored. This method will always return max long.
+     *
      * {@inheritDoc}
      */
     @Override
     public long getTotalSpace() throws IOException {
-        return 0;
+        return Long.MAX_VALUE;
```

Comment:
Would `-1` or a public constant be a better value for this to have clear indication that this is a unique value and shouldn't be used to determine code paths or in validation.

Reply:
I imagine there might be scenarios where customers might try to compare the amount of space in a store with the (possibly predicted) amount of data they have. In that case, a `getTotalSpace() > dataSize` check would always fail. Just speculation, though. Thoughts?